ocaml/stdlib/float_u.ml

```diff
@@ -0,0 +1,156 @@
+# 1 "float_u.ml"
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                        Nicolas Ojeda Bar, LexiFi                       *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+open! Stdlib
+
+[@@@ocaml.flambda_o3]
+
+
+external to_float : float# -> (float[@local_opt]) = "%box_float"
+
+external of_float : (float[@local_opt]) -> float# = "%unbox_float"
+
+(* CR layouts: Investigate whether it's worth making these things externals.
```

Comment:
We have an internal ticket to come back to this point. Not sure the CR is helping beyond that.

Reply:
I think in a situation like this, where the CR is related to a natural question someone might have reading the code, it's nice to have here.  But I can change it from a CR to a regular comment if you prefer.

Reply:
No, it's fine. You've identified how it's helping.